src/App.tsx

```diff
@@ -1,61 +1,112 @@
+import { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { User } from './types/User';
+import { PreparedTodo } from './types/PreparedTodo';
+
+function getUserById(userId: number) {
+  const user = usersFromServer.find(userTodo => userTodo.id === userId);
+
+  return user !== null ? user : undefined;
+}
+
+const initialTodos = todosFromServer.map(todo => ({
```

Comment:
```suggestion
const initialTodos = todosFromServer.map(todo => ({
```
is it initial? looks like its todosWithUser?